Sources/SwiftParser/Expressions.swift

```diff
@@ -907,7 +907,9 @@ extension Parser {
       at: openDelimiter != nil ? .leadingRaw : .leading,
       text: text
     ) ?? RawTokenSyntax(missing: .stringQuote, arena: arena)
-    text = text.dropFirst(openQuote.tokenText.count)
+    if !openQuote.isMissing {
```

Comment:
I fixed logic here.

If opener quote missing token is generated, don't drop text.